figures/3.figure3/scripts/3.assemble_montage.r

```diff
@@ -442,7 +443,7 @@ hist_plot_apoptosis <- (
 )
 
 hist_plot_pyroptosis <- (
-    ggplot(pyroptosis_df, aes(x = Cytoplasm_RadialDistribution_ZernikePhase_CorrGasdermin_9_1))
+    ggplot(pyroptosis_df, aes(x = Cytoplasm_RadialDistribution_ZernikeMagnitude_CorrER_9_9))
```

Comment:
Just curious, why was the feature changed here and in the rest of the file for pyroptosis?

Reply:
After the rerun the feature selection changed so I had to change some features.